src/libraries/pkg/baseline/packageIndex.json

```diff
@@ -6245,8 +6245,12 @@
       }
     },
     "System.Speech": {
+      "BaselineVersion": "5.0.0",
       "InboxOn": {
         "net45": "4.0.0.0"
+      },
+      "AssemblyVersionInPackageVersion": {
+        "4.0.0.0": "5.0.2"
```

Comment:
Is it correct that this is 5.0.2? If I use 5.0.0 I get an error: `PackageIndex from C:\git\runtime\src\libraries\pkg\baseline\packageIndex.json indicates that assembly version 4.0.0.0 is contained in non-stable package version 5.0.0 which differs from this package version 5.0.2`

I'm not sure where 5.0.2 comes from, as eg., System.Diagnostics.EventLog has 5.0.0.

Reply:
You should list the first package version that this assembly version appeared in.  This should be 5.0.0

Reply:
That was my understanding but if I try
```
    "System.Speech": {
      "BaselineVersion": "5.0.0",
      "InboxOn": {
        "net45": "4.0.0.0"
      },
      "AssemblyVersionInPackageVersion": {
        "4.0.0.0": "5.0.0"
      }
    },
```
then I get 
```
Packaging.targets(1112,5): error : PackageIndex from C:\git\runtime\src\libraries\pkg\baseline\packageIndex.json indicates that assembly version 4.0.0.0 is contained in non-stable package version 5.0.0 which differs from this package version 5.0.2. [C:\git\runtime\src\libraries\System.Speech\pkg\System.Speech.pkgproj]
```
I'm not sure where 5.0.2 is coming from.

Reply:
Please set PackageVersion to 5.0.0 in Directory.Build.props for this library.  Normally we set PackageVersion in servicing for libraries, otherwise they'll default to the current ProductVersion which increments in servicing.  This package is special since you're shipping it for the first time in servicing.